docs/conf.py

```diff
@@ -124,11 +124,14 @@ def setup(sphinx):
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 html_theme = 'sphinx_rtd_theme'
-
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-#html_theme_options = {}
+html_theme_options = {
+    'logo_only': True,
+    'style_nav_header_background': '#65afff',
+    'display_version': True,
```

Comment:
> Was unable to resolve the **version issue where 'v' was missing**. Seems to be existing. I think we need to pass the version as v0.8.8 in the version variable (which I think is the **CMakeLists.txt**)  file.

I think that version gets overridden by ReadTheDocs. If you build docs locally on develop you don't get the `v` either. And ReadTheDocs can also display something completely different there - for it's `latest` for https://docs.soliditylang.org/en/latest/ and `breaking` for https://docs.soliditylang.org/en/breaking/ even though these do have 0.8.8 in `CMakeLists.txt`.

I think that the `v` will be there when the PR gets merged and docs get deployed.